src/math/numbers.c

```diff
@@ -6,25 +6,59 @@
 //         Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
 
-/* Euclidean algorithm for greatest common denominator from
- * pseudocode in
- * https://en.wikipedia.org/wiki/Euclidean_algorithm#Implementations
- */
-
 #include <sof/audio/format.h>
 #include <sof/math/numbers.h>
 #include <stdint.h>
 
+/* This functions returns the greatest common divisor of two numbers
+ * If both parameters are 0, gcd(0, 0) returns 0
+ * If first parameters is 0 or second parameter is 0, gcd(0, b) returns b
+ * and gcd(a, 0) returns a, because everything devides 0.
+ * Link to algorithm:
```

Comment:
I would add the link to algorithm at the beginning of the file. Like it was until now.